generators/PleadingKitten/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Generator } from '../../types/GeneratorTypes';
+import { SettingType } from '../../types/SettingTypes';
+import { generate } from './generate';
+import { PleadingKittenSettings } from './types';
+
+export const pleadingKittenGenerator: Generator<PleadingKittenSettings> = {
+  name: 'Pleading kitten',
+  description: 'Two cats. One gets all the attention.',
+  helpText: 'Enter labels for the top and bottom cats.',
+  generate,
+  settings: {
+    topCatLabel: {
+      name: 'Top Cat',
```

Comment:
Very minor but I tend to use sentence case for the setting names

```suggestion
      name: 'Top cat',
```